go.mod

```diff
@@ -24,6 +24,7 @@ require (
 	github.com/poy/onpar v1.1.2 // indirect
 	github.com/prometheus/client_golang v1.8.0
 	github.com/rakyll/statik v0.1.7
+	github.com/regen-network/cosmos-proto v0.3.1 // indirect
```

Comment:
Run `go mod tidy` may remove this dependency.

Reply:
6f1537f